static/app/styles/prism.tsx

```diff
@@ -106,13 +108,11 @@ export const prismStyles = (theme: Theme) => css`
     }
     .line-highlight {
       position: absolute;
-      left: 0;
+      left: -${space(2)};
       right: 0;
-      padding: inherit 0;
-      margin-top: 1em;
       background: var(--prism-highlight-background);
       box-shadow: inset 5px 0 0 var(--prism-highlight-accent);
-      z-index: 0;
+      z-index: -1;
```

Comment:
With `z-index: 0` the line highlight overlapped the text

Reply:
Nice, thanks for getting the right hex codes! Changed the color variables to this